static/scss/pages/settings.scss

```diff
@@ -0,0 +1,120 @@
+#settings-main {
+
+    .c-settings-headline {
+        @include text-title-2xs;
+        padding: $spacing-lg $spacing-lg $spacing-md ;
+        color: $color-grey-60;
+    }
+
+    .c-meta--links {
+        display: none;
+    }
+
+    .c-settings-form--heading {
+        @include text-title-3xs;
+        color: $color-grey-50;
+        font-weight: normal;
+    }
+
+    .c-settings-form--footer {
+        background-color: transparent;
+        text-align: center;
+        display: block;
+    }
+
+    .c-settings-form--field {
+        border-bottom: 1px solid $color-grey-10;
+    }
+
+    .c-settings-form--inline-warning {
+        box-shadow: none;
+        text-align: start;
+        margin-left: 0;
+    }
+
+    .c-settings-form--submit {
+        padding: $spacing-sm $spacing-2xl;
+    }
+
+    .c-settings-form--notification {
```

Comment:
Info: I wanted to call this out here, but @codemist may have already tagged you. She also created the [same settings-notification](https://github.com/mozilla/fx-private-relay/pull/1154) (visible on the profile page, with some additional functionality). Could you two sync up your code here? That may mean moving it out of this partial entirely.

Reply:
ref: https://github.com/mozilla/fx-private-relay/pull/1154

Reply:
@maxxcrawford for my PR, the scss code is in `main.scss` right now. Would it be better fit in `settings.scss`?

Reply:
@codemist @Vinnl No – It needs to be outside of both. Most likely in the `components/banners` SCSS file.

Reply:
Oh nice, and yours is far more complete than mine! I've submitted a PR to extract that into a separate component that we can reuse, maybe we can collaborate on that and reuse it in our respective PRs? Or maybe it's a completely crazy approach and we better just use copy-paste the HTML in the different locations :)

https://github.com/mozilla/fx-private-relay/pull/1161

Reply:
Let's work on reusing it in that PR, and replace the code here with whatever we land on there after this is merged.